app/views/kudo_mailer/batch_kudo_notification.html.erb

```diff
@@ -1,19 +1,29 @@
 <% content_for :message do %>
 
-    <% @commentables.each_with_index do |commentable, index| %>
-        <%
-           givers_list = @kudo_givers["#{commentable.class.name}_#{commentable.id}"].
-             map { |k| style_bold(k) }.
-             to_sentence.html_safe
-           commentable_link = style_creation_link(commentable.commentable_name.html_safe, polymorphic_url(commentable))
-        %>
+  <% @commentables.each_with_index do |commentable, index| %>
+    <%
+      commentable_link = style_creation_link(commentable.commentable_name, polymorphic_url(commentable))
+      givers_hash = @user_kudos["#{commentable.class.name}_#{commentable.id}"]
+      # dup so we don't add "a guest" or "5 guests" to the original array. If
+      # that happens, kudo_count ends up too high whenever both named and guest
+      # kudos are present.
+      names = givers_hash["names"].dup
+      guest_count = givers_hash["guest_count"].to_i
+      kudo_count = names.size + guest_count
+      givers = names
```

Comment:
This makes sense but confuses me a little bit at a glance. I think it's clearer if you have:
```rb
names = givers_hash["names"]
# more code here

# dup so we don't add "a guest" or "5 guests" to the original array. [...]
givers = names.dup
```